.env

```diff
@@ -0,0 +1,4 @@
+MASTER_KEY=123
+HOST=http://127.0.0.1:7700
+PRIVATE_KEY=8c222193c4dff5a19689d637416820bc623375f2ad4c31a2e3a76e8f4c70440d
+PUBLIC_KEY=948413b6667024a0704c2023916c21eaf0a13485a586c43e4d2df520852a4fb8
```

Comment:
I use `dotenv` to set the keys in the .env file and import them in my jest setup. This should become only possible in development mode. 
Later there will be a condition on the mode (test or dev)and based on that it will take the information from .env or directly from env variables (in testing mode in github actions).

Reply:
Why don't you just hang some static variables for testing?